cmd/apply/main.go

```diff
@@ -104,7 +105,11 @@ func run(_ *cobra.Command, args []string) {
 	}
 
 	promptAnswer := false
-	fileInfo, _ := os.Stdin.Stat()
+	fileInfo, err := os.Stdin.Stat()
+	if err != nil {
+		log.Fatal(err.Error())
+	}
```

Comment:
I am not sure if we actually care about the errors here.

Reply:
Since we're checking for an error, it certainly doesn't hurt to have it here.

If it turns out to be an issue, you can always remove it in the future.